lib/src/excel.dart

```diff
@@ -35,49 +34,65 @@ class ExcelDart {
   static const _resultTableColumnStartAddress = 'A';
   static const _resultTableRowStartAddress = 4;
 
-  static const _fontName = 'Roboto';
-  static const _fontSize = 12;
-
   static const _centerAlignment = 'Center';
-  static const _rightAlignment = 'Right';
-  static const _leftAlignment = 'Left';
 
   static const _borderStyle = 'Continuous';
+  static const _tableStyle = 'TableStyleLight9';
+
   static const _currencyFormat = '\$#,##0.00';
   static const _timeFormat = '12:00:00 AM';
 
   /// Column addresses calculated based on [_tableHeader].
   static final _currentPacingColumnAddress =
-      '\$${_getExcelColumnReference(_tableHeader.indexOf('Current Pacing %'))}'
-      '${_resultTableRowStartAddress + 1}';
-  static final _budgetUnitColumnHeaderAddress =
-      '\$${_getExcelColumnReference(_tableHeader.indexOf('Budget Unit'))}'
+      '\$${_getExcelColumnReference(_tableHeader.indexOf(_currentPacingColumnName))}'
+      '${_resultTableRowStartAddress}';
+  static final _budgetUnitColumnAddress =
+      '\$${_getExcelColumnReference(_tableHeader.indexOf(_budgetUnitColumnName))}'
       '${_resultTableRowStartAddress}';
 
+  /// Table header names (spaces are not allowed).
+  static const _displayNameColumnName = 'Display_Name';
+  static const _advertiserIdColumnName = 'Advertiser_ID';
+  static const _campaignIdColumnName = 'Campaign_ID';
+  static const _insertionOrderIdColumnName = 'Insertion_Order_ID';
+  static const _entityStatusColumnName = 'Entity_Status';
+  static const _budgetUnitColumnName = 'Budget_Unit';
+  static const _budgetColumnName = 'Budget_Amount';
+  static const _spentColumnName = 'Spent_Amount';
+  static const _currentPacingColumnName = 'Current_Pacing_%';
+  static const _pacingPeriodColumnName = 'Pacing_Period';
+  static const _pacingTypeColumnName = 'Pacing_Type';
+  static const _startDateColumnName = 'Start_Date';
+  static const _endDateColumnName = 'End_Date';
+  static const _dailyMaxColumnName = 'Daily_Max';
+  static const _dailyMaxImpressionColumnName = 'Daily_Max_Impression';
```

Comment:
Do you need to split this into 2 cols - you have budget unit. Aim to be consistent ie either split budget amount col into 2 by type or merge these 2 into 1 col. I prefer the later.

Reply:
Sounds, I will address this in a separate pull request since I'll need to change the proto and some other tests.

New issue created: #68